docs/How-to-test-the-application.md

```diff
@@ -0,0 +1,59 @@
+# How to test App:
+- Using Physical device and apk
+- Using Emulator and Apk
+- Directly Running in Android Studio ( Recommended )
+
+
+## For option one or two:
+
+Every time a new commit is merged into the `develop` branch an _apk_ is released in the `apk` branch. To Navigate to `apk` Branch see here: 
+
+![Branch chnage](https://user-images.githubusercontent.com/16835617/92631953-8a52a900-f2ef-11ea-9e61-60692555283b.png)
+
+
+If you want to do manual testing on the application without setting up the development environment, you can download this [app-debug.apk](https://github.com/anitab-org/mentorship-android/blob/apk/app-debug.apk) and install it on your Android device.
+
+![apk-branch](https://user-images.githubusercontent.com/16835617/97575784-42105700-1a13-11eb-8a73-4669b8ebb343.png)
+
+
+![apkInstall](https://user-images.githubusercontent.com/16835617/97575669-1db47a80-1a13-11eb-8074-bcac49ee59db.png)
+
+
+For Testing on a physical device tick the option here :
+![Screenshot of Unknown install option ](https://static.apkpure.com/www/static/imgs/unknown_sources.jpg)
+
+Now click the apk to install !
+
+## For Testing on Emulator with APk :
+
+1. Now Open the project in Android Studio
+2. Wait untill all background process are finished 
+3. click the AVD button or click from already created devices in Android studio click 
```

Comment:
also can you please make following changes in line 31 : `click on the AVD button or select from` and remove the extra click at the end of the line :-) ?